frontend/src/app/patients/UploadPatients.tsx

```diff
@@ -50,25 +59,42 @@ const UploadPatients = ({ isAdmin }: { isAdmin: boolean }) => {
       }
     };
   }
+
+  function createErrorToast(
+    header: string,
+    body: string,
+    includeGuide: boolean
+  ) {
+    setErrorMessageHeader(header);
+    setErrorMessageText(body);
+    setIncludeGuideInErrorMessage(includeGuide);
```

Comment:
Do these three states really need to be separate states?. Could a single state with a more specific data structure work as well? For example:

type ErrorMessage = {
text: string,
header?:string,
includeGuide:boolean
}

This will allow to save some renders

Reply:
Thank you for the suggestion! I like this much better, changed to a single `errorMessage` state.